addons/l10n_hr_edi/demo/demo_company.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <function model="res.company" name="write">
+        <value eval="[ref('base.demo_company_hr')]"/>
+        <value eval="{'l10n_hr_mer_username': '12513', 'l10n_hr_mer_password': 'AeXLoJf9Ld', 'l10n_hr_mer_company_id': 'BE0477472701', 'l10n_hr_mer_software_id': 'Test-002'}"/>
+    </function>
+
+    <function model="res.partner" name="write">
+        <value eval="[ref('base.partner_demo_customer_hr')]"/>
+        <value eval="{'invoice_edi_format': 'ubl_hr'}"/>
+    </function>
```

Comment:
Let's use `<record>` to update these existing records (unless there's a good reason not to)